src/shared/components/SkeletonLoader/SkeletonLoader.tsx

```diff
@@ -65,12 +49,12 @@ const SkeletonLoaderAnimated = styled.div<SkeletonLoaderProps>`
   transform: translateX(0%);
   background: linear-gradient(
     104deg,
-    rgba(183, 200, 250, 0) 15%,
-    rgba(183, 200, 250, 0.06) 30%,
-    rgba(187, 217, 246, 0.13) 48%,
-    rgba(187, 217, 246, 0.13) 52%,
-    rgba(183, 200, 250, 0.06) 70%,
-    rgba(183, 200, 250, 0) 85%
+    rgba(0, 0, 0, 0) 15%,
```

Comment:
Why have you changes this? According to this comment: https://github.com/Joystream/atlas/pull/1492#issuecomment-952601955
It should be `rgba(183, 200, 250, 0) 15%`

Reply:
So the starting color has 0 opacity which makes it fully transparent. Bartosz did testing and turns out it doesn't matter what the base color is, if the opacity is 0, the transition will look the same. Here are the test results (one is transparent black, the other `rgba(183, 200, 250, 0)` :
![Clipboard - 4 listopada 2021 12_47](https://user-images.githubusercontent.com/12646744/140376236-86b6fc64-ffd4-4bc6-972e-30ebbdc6aa0b.png)